engine/src/main/scala/cromwell/webservice/routes/wes/WesRouteSupport.scala

```diff
@@ -49,36 +56,53 @@ trait WesRouteSupport extends HttpInstrumentation {
         pathPrefix("ga4gh" / "wes" / "v1") {
           concat(
             path("service-info") {
-              complete(ServiceInfo.toWesResponse(workflowStoreActor))
+              get {
+                complete(ServiceInfo.toWesResponse(workflowStoreActor))
+              }
             },
-            pathPrefix("runs") {
```

Comment:
I see you've removed this path prefix and added a `"runs"` to the start of all the cases. That's fine, just curious whether it was a deliberate choice?

Reply:
+1 on curiosity, I do think `pathPrefix` is more conventional to reduce repetition, but I'm sure this works fine too

Reply:
I originally intended to use `pathPrefix` for repetitions sake, but doing so confused all of the Routes for some reason, and the fix I found was to simply give all the routes their own `runs`.